cl/audio/factories.py

```diff
@@ -0,0 +1,25 @@
+from factory import Faker
+from factory.django import DjangoModelFactory, FileField
+
+from cl.audio.models import Audio
+from cl.search.factories import DocketParentMixin
+from cl.search.models import SOURCES
+
+
+class AudioFactory(DjangoModelFactory):
+    class Meta:
+        model = Audio
+
+    id = Faker("random_id")
+    source = Faker("random_element", elements=SOURCES)
```

Comment:
I'd be surprised if this were working properly. Could you verify if you haven't already? I think you need something like this instead:

    source = FuzzyChoice(Docket.SOURCE_CHOICES, getter=lambda c: c[0])

...or at least *something* that picks `c` instead of `('c', "court")` (for example).

Reply:
√.  I dont think the value is docket.source_choices but yes ... you are right.